Scripts/Runtime/Entities/Tasks/Data/JobTaskWorkData.cs

```diff
@@ -0,0 +1,12 @@
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// An <see cref="AbstractTaskWorkData"/> specific for use in scheduling Jobs
+    /// </summary>
+    public class JobTaskWorkData : AbstractTaskWorkData
+    {
+        internal JobTaskWorkData(AbstractTaskDriverSystem system) : base(system)
```

Comment:
Ok, yea, this depends on what we end up doing with the MainThreadTaskWorkConfig. If it's all rolled into one I think we'd be better off with just a single `TaskWorkConfig` since it's agnostic to the execution type.

Reply:
Let's chat.

Reply:
Sorted out in other thread. Resolving